content/en/docs/tasks/configure-pod-container/configure-service-account.md

```diff
@@ -21,10 +21,13 @@ customized the behavior in your cluster, in which case this documentation may
 not apply.
 {{< /note >}}
 
-When you (a human) access the cluster (for example, using `kubectl`), you are
-authenticated by the apiserver as a particular User Account (currently this is
-usually `admin`, unless your cluster administrator has customized your cluster). Processes in containers inside pods can also contact the apiserver.
-When they do, they are authenticated as a particular Service Account (for example, `default`).
+When you, a human, access the cluster (for example, using `kubectl`) you are
+authenticated by the apiserver as a particular user identity. For example, you
```

Comment:
- Do you need, `a human`?
When you access a cluster using a command line tool, such as `kubectl`, the API server authenticates the request using a specific user account (currently ... ).
- user identity vs user account, what is the difference?

Reply:
- dropped “a human”
- I'm using the term “identity” and the page doesn't mention “user account” anywhere anymore (unless I missed one)